pkg/query-service/model/config.go

```diff
@@ -0,0 +1,57 @@
+package model
+
+import (
+	"os"
+
+	"gopkg.in/yaml.v2"
+)
+
+type SkipConfig struct {
+	Services []ServiceSkipConfig `yaml:"services"`
+}
+
+type ServiceSkipConfig struct {
+	Name       string   `yaml:"name"`
+	Operations []string `yaml:"operations"`
+}
+
+func (s *SkipConfig) ShouldSkip(serviceName, name string) bool {
```

Comment:
does it make more sense to use array vs map in reading from `yaml` config? Otherwise map would have been more efficient. It scares me when I see 2 for loops in a function. The good thing is we are not expecting perf hits as arrays would be very small. From DS perspective, 2 for loops are bad.

Reply:
Yes, it doesn't change anything significant whether we use maps or slices when the number of items is very small.